aesara/tensor/elemwise.py

```diff
@@ -1334,26 +1325,8 @@ def __init__(self, scalar_op, axis=None):
         self.set_ufunc(scalar_op)
 
     def set_ufunc(self, scalar_op):
-        # TODO FIXME: Why would we ever do this, instead of allowing the `Op`
-        # itself to tell us which `ufunc` it should use?
-        if isinstance(scalar_op, Add):
-            self.ufunc = np.add
-        elif isinstance(scalar_op, Mul):
-            self.ufunc = np.multiply
-        elif isinstance(scalar_op, ScalarMaximum):
-            self.ufunc = np.maximum
-        elif isinstance(scalar_op, ScalarMinimum):
-            self.ufunc = np.minimum
-        elif isinstance(scalar_op, AND) and _numpy_ver >= [1, 12]:
-            # numpy.bitwise_and.identity was incorrect for versions before
-            # 1.12 (it was 1 instead of -1), so we skip it in that case.
-            # We will fall back to the "else:" case, which defines a
-            # ufunc without identity.
-            self.ufunc = np.bitwise_and
-        elif isinstance(scalar_op, OR):
-            self.ufunc = np.bitwise_or
-        elif isinstance(scalar_op, XOR):
-            self.ufunc = np.bitwise_xor
+        if hasattr(scalar_op, "nfunc_spec") and hasattr(np, scalar_op.nfunc_spec[0]):
+            self.ufunc = getattr(np, scalar_op.nfunc_spec[0])
```

Comment:
Here I assume `ScalarOp`'s `nfunc_spec` is used for getting the function as attribute from Numpy. Similar to how `ScalarOp`'s `nfunc_variadic` is used for getting the function as attribute from Jax.